jdi-eyes-demo/src/main/java/jdisite/custom/MultiSelect.java

```diff
@@ -1,24 +1,30 @@
 package jdisite.custom;
 
-import static com.epam.jdi.light.logger.LogLevels.DEBUG;
-import static com.epam.jdi.tools.LinqUtils.map;
-import static org.jsoup.internal.StringUtil.isBlank;
-
 import com.epam.jdi.light.asserts.generic.UISelectAssert;
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.elements.base.UIListBase;
 import com.epam.jdi.light.elements.common.UIElement;
 import com.epam.jdi.light.elements.complex.WebList;
 import com.epam.jdi.light.elements.interfaces.base.HasLabel;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+
 import java.util.List;
 
-public class MultiSelect extends UIListBase<UISelectAssert<?,?>> implements HasLabel {
-    @UI(".caret") UIElement expander;
-    @UI("ul") UIElement valuesList;
-    @UI("button") UIElement valueText;
-    @UI("li") WebList allValues;
-    @UI("input") WebList value;
+import static com.epam.jdi.light.logger.LogLevels.DEBUG;
+import static com.epam.jdi.tools.LinqUtils.map;
+import static org.jsoup.internal.StringUtil.isBlank;
+
+public class MultiSelect extends UIListBase<UISelectAssert<?, ?>> implements HasLabel {
+    @UI(".caret")
+    UIElement expander;
+    @UI("ul")
+    UIElement valuesList;
+    @UI("button")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)